src/cdn_definitions/schema.yaml

```diff
@@ -11,7 +11,211 @@ $id: http://release-engineering.github.io/cdn-definitions/schema.json
 
 
 definitions:
+  arch:
+    type: string
+    pattern: "^(x86|x86\\_64|ia64|i386|ppc|ppc64|ppc64le|s390|s390x|aarch64)$"
```

Comment:
I think this doesn't belong in the schema.  At first I thought it was borderline, but then I thought of this scenario:

"We're going to start shipping products for <new_arch>, this is company confidential until <some_marketing_event> on which date it will be both announced and released"

We would be forced to add the new secret arch into the public schema which wouldn't be acceptable.

Note: if you feel it's worthwhile to verify that arch values fall into some expected range, this could be an example of the kind of additional test beyond the schema which makes sense to add in cdn-definitions-private.